mimic/rest/maas_api.py

```diff
@@ -276,7 +282,15 @@ def create_multiplot_from_metric(metric, reqargs, allchecks):
                     d = {}
                     d['numPoints'] = 4
                     d['timestamp'] = timestamp
-                    d['average'] = random.randint(1, 99)
+                    if not q % (points / 4):
+                        squarewave_downtrend = squarewave_downtrend ^ 1
```

Comment:
minor nitpick: this can be more succinctly expressed as `squarewave_downtrend ^= 1`.